corehq/apps/accounting/bootstrap/config/submitting_mobile_worker_feature_rate.py

```diff
@@ -0,0 +1,8 @@
+from decimal import Decimal
+from corehq.apps.accounting.models import FeatureType
+
+BOOTSTRAP_CONFIG = {
+    "feature_rates": {
+        FeatureType.SUBMITTING_MOBILE_WORKER: dict(monthly_limit=2000, per_excess_fee=Decimal('3.00'))
```

Comment:
I realize this is possibly the most annoying type of feedback but—would you be open to changing the canonical name of this in both code and translated human text from "submitting mobile worker" to "form-submitting mobile worker" (`form_submitting_mobile_worker`, `FORM_SUBMITTING_MOBILE_WORKER`, `"Form-Submitting Mobile Worker"`, etc.)? I realize it's kinda long, but I think it's worth being consistent (currently both are used in this code, and form-submitting is what we've been saying out loud to each other) and form-submitting is more explicit/harder to be confused about what it means.

Reply:
I agree it's clearer, was just worried about being overly verbose. I can make that change.

Reply:
I had originally suggested "Submitting Mobile Worker" as a replacement for "Active User" which—also leaning to something less verbose but still more descriptive. However, I second Danny's suggestion here to be more verbose and err on the side of extra clarity for the longer term :)

Reply:
What do we think about `DomainSubmittingMobileWorkerHistory`? Does that become `DomainFormSubmittingMobileWorkerHistory` in the change as well, or can it possibly drop `Domain` since that's implied by inheriting from `DomainUserHistoryBase` now?

Reply:
I think it's fine to drop the `Domain` in the name. In favor of `FormSubmittingMobileWorkerHistory`

Reply:
https://github.com/dimagi/commcare-hq/pull/34847/commits/6d70f3fa19028fbf414d08cbd87f5ceb884e4032